### PR TITLE
Add ShiftAndCapsLock

### PR DESCRIPTION
**Pull request by K0baU**
I added a new KeyAction called ShiftAndCapsLock. It does the same thing as ToggleShiftMode>ToggleCapsLock.

### CODE REVIEW

File: app/src/main/java/com/dessalines/thumbkey/keyboards/JAKanaThumbKey.kt [hunk 1]
```diff
@@ -105,7 +105,7 @@ val KB_JA_KANA_THUMBKEY_HIRAGANA =
                     center =
                         KeyC(
                             display = KeyDisplay.TextDisplay("ア"),
-                            action = ToggleCapsLock,
+                            action = ShiftAndCapsLock(true),
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/JAKanaThumbKey.kt:108]: If enable is always true, you might as well remove it. Unless you plan on adding another one that disables shift mode.

File: app/src/main/java/com/dessalines/thumbkey/utils/Utils.kt [hunk 2]
```diff
@@ -892,6 +892,12 @@ fun performKeyAction(
         }
 
         KeyAction.ToggleCapsLock -> onToggleCapsLock()
+        is KeyAction.ShiftAndCapsLock -> {
+            val enable = action.enable
+            Log.d(TAG, "Toggling Shifted: $enable")
+            onToggleShiftMode(enable)
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/utils/Utils.kt:898]: Can just make this true.

**Review by dessalines — CHANGES_REQUESTED** [review]
See comments above.

**Comment by K0baU** [comment]
I added ShiftAndCapsLock(false) in line 233.

**Review by dessalines — APPROVED** [review]

**Pull request merged**